ggrasp_service:
  model: ggcnn_l1_smooth_0.5_iou_0.97
  camera:
    info_topic: /camera/depth/camera_info
    depth_topic: /camera/depth/image_meters
    camera_frame: camera_depth_optical_frame
    robot_base_frame: panda_link0
    crop_size: 300
    crop_y_offset: 40
    fov: 65.5
